packages/ui-toolkit/src/base/global.js

```diff
@@ -44,6 +44,10 @@ export default ({ theme }) => css`
     flex-flow: column;
   }
 
+  form {
+    margin: 0;
```

Comment:
I think browsers render form with margin 0 by default. So, if margins aren't set anywhere else I guess this is unnecessary ;)

Reply:
I had to do the same in my branch, so I think at least chrome is changing the margin of form.

Reply:
@ramitos is right @fabiommmoreira. Chrome is adding a 10px margin on the bottom of `form` elements.